must-gather/collection-scripts/gather

```diff
@@ -43,9 +43,10 @@ commands+=("get sc")
 commands+=("get infrastructures.config -oyaml")
 
 # Run the Collection of Resources to list
-for command in "${commands[@]}"; do
-     echo "collecting oc command ${command}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
-     timeout 120 oc ${command}  >> ${BASE_COLLECTION_PATH}/oc_output/"${command}"
+for ((i = 0; i < ${#commands[@]}; i++)); do
+     echo "collecting oc command ${commands[$i]}"
+     COMMAND_OUTPUT_FILE=oc_output/${commands[$i]// /_}
```

Comment:
this commands *get pods -owide -n openshift-storage* is getting executed like this get_pods_-owide_-n_openshift-storage:  same is happening with every command.